store/store.go

```diff
@@ -484,7 +534,7 @@ func (c *MongoStoreClient) GetDeviceData(p *Params) (StorageIterator, error) {
 
 	// _schemaVersion is still in the list of fields to remove. Although we don't query for it, data can still exist for it
 	// until BACK-1281 is done.
-	removeFieldsForReturn := bson.M{"_id": 0, "_userId": 0, "_groupId": 0, "_version": 0, "_active": 0, "_schemaVersion": 0, "createdTime": 0, "modifiedTime": 0}
+	removeFieldsForReturn := bson.M{"_id": 0, "_userId": 0, "_groupId": 0, "_version": 0, "_active": 0, "_schemaVersion": 0, "createdTime": 0, "modifiedTime": 0, "_deduplicator": 0}
```

Comment:
Is this change intended?

Reply:
yes, when testing the clients, tide-whisperer was leaving extra properties in the return, likely just forgotten, seemed like some minor cleanup.
It can be reverted, if needed. Its an internal property that shouldnt have been hitting clients from what i can tell.

Reply:
Although it's internal, there might be 3rd party clients which depend on it.